src/main/java/net/earthcomputer/clientcommands/command/AutoPrefixCommand.java

```diff
@@ -0,0 +1,60 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.commands.CommandBuildContext;
+import net.minecraft.network.chat.Component;
+
+import static dev.xpple.clientarguments.arguments.CComponentArgument.getComponent;
+import static dev.xpple.clientarguments.arguments.CComponentArgument.textComponent;
+import static dev.xpple.clientarguments.arguments.CGameProfileArgument.gameProfile;
+import static dev.xpple.clientarguments.arguments.CGameProfileArgument.getSingleProfileArgument;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.argument;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class AutoPrefixCommand {
+    private static String currentPrefix = null;
```

Comment:
I feel like this entire command can be a config instead, but I will continue reviewing the command anyway.

Reply:
should it be a config? Should this be persistent?

Reply:
configs can be temporary

Reply:
Temporary configs exist. Although you would lose the autocompletion for players, unless you resort to a hack. I'll let @Earthcomputer decide.

Reply:
ah, i understand, you meant making this a config setting qwq *cries*

Reply:
Yes please move it to a config

Reply:
Done, haven't tested it tho